File: OOP/Seminars/HomeWork/Seminar_7/PY/ClientData.py
from Client import *
class ClientData:
    def __init__(self):
        self.clients = []
        self.count = 1

    def get_clients(self):
        return self.clients

    def add_client(self, Client):
        self.clients.append(Client)
        Client.set_id(self.count)
        self.count += 1

    def remove_client(self, client):
        self.clients.remove(client)

    def show_clients(self):
        print(self.clients)

    def client_change_balance(self, client_id, value):
        for client in self.clients:
            if client.get_id() == client_id:
                client.set_balance(client.get_balance() + value)

File: OOP/Seminars/HomeWork/Seminar_7/Java/dataBase/ClientData.java
package OOP.Seminars.HomeWork.Seminar_7.Java.dataBase;

import OOP.Seminars.HomeWork.Seminar_7.Java.model.Client;

import java.util.ArrayList;
import java.util.List;

public class ClientData {
    private List<Client> clients = new ArrayList<>();
    private int count=1;
    public List<Client> getClients() {
        return clients;
    }
    public void addClient(Client client){
        clients.add(client);
        client.setId(count);
        count++;
    }
    public void removeClient(Client client){
        clients.remove(client);
    }
    public void showClients(){
        System.out.println(clients);
    }
    public void clientChangeBalance(int id, int value){
        for (Client client : clients){
            if(client.getId()==id){
                client.setBalance(client.getBalance()+value);
            }
        }
    };
}